R/diffdf.R

```diff
@@ -89,7 +90,8 @@ diffdf <- function(
     strict_factor = TRUE,
     file = NULL,
     tolerance = sqrt(.Machine$double.eps),
-    scale = NULL
+    scale = NULL,
+    check_column_order = TRUE
```

Comment:
I think this should be false by default, otherwise we are not backwards compatible, and I think minor releases should be backwards compatible